client/ayon_nuke/api/plugin.py

```diff
@@ -1225,3 +1247,31 @@ def exposed_write_knobs(settings, plugin_name, instance_node):
         instance_node.addKnob(nuke.Text_Knob('', 'Write Knobs'))
     write_node = nuke.allNodes(group=instance_node, filter="Write")[0]
     link_knobs(exposed_knobs, write_node, instance_node)
+
+
+def update_write_node_filepath(created_inst, changes):
```

Comment:
Please add a docstring.
```suggestion
def update_write_node_filepath(created_inst, changes):
    """Update instance node on context changes.
    
    Whenever any of productName, folderPath, task or productType
    changes then update:
    - output filepath of the write node
    - instance node's name to the product name
    """
```